ansible/roles/girder/tasks/main.yml

```diff
@@ -101,6 +105,15 @@
     state: present
   sudo: yes
 
+
+- name: Create directory for build
+  file: state=directory path="{{ root_dir }}/girder/clients/web/static/built"
+
+- name: Download fontello.zip
+  command: wget -O fontello.zip https://data.kitware.com/api/v1/file/57c5d1fc8d777f10f269dece/download
```

Comment:
Yes.  grunt-curl won't redownload a file if it exists (which is part of the problem, since if it gets a partial download you have to manually delete the file to allow it to try again).

We run `girder-install web` in ansible/roles/dev/tasks/main.yml.